docs/repositories/issues.md

```diff
@@ -7,29 +7,36 @@ By default, the page lists the issues on the main branch of your repository but
 !!! note
     [You can use the Codacy API](../codacy-api/examples/obtaining-current-issues-in-repositories.md) to generate reports or obtain information about the current issues in your repositories in a more flexible way.
 
-![Issues page](images/issues.png)
+![Issues page](images/issues.png)<!-- TODO CY-6604 Update screenshot -->
 
 <!--issue-detail-start-->
 Click the title of an issue to see the following information:
 
--   The committer and date of the commit that introduced the issue
--   The [tool that reported the issue](../getting-started/supported-languages-and-tools.md) and the estimated time to fix it
--   What's the issue and how to solve it
--   The programming language and category of the issue
+-   The committer and date of the commit that introduced the issue<!-- TODO CY-6604 I'm not seeing this. Is this by design? -->
```

Comment:
I'm not seeing the committer's name and commit date here. Is this by design?

Reply:
I think it’s not available for all commits, and there’s not UI hint that it’s missing as it did before

Reply:
here's an example:
https://app.codacy.com/p/298551/issues/index?bid=16052413&resultDataId=131052218713
https://app.codacy.com/gh/codacy/codacy-spa/issues/131052218713
![Uploading Screenshot 2023-10-11 at 13.38.53.png…]()